sdk/src/beta9/abstractions/endpoint.py

```diff
@@ -208,6 +209,9 @@ def local(self, *args, **kwargs) -> Any:
 
     @with_grpc_error_handling
     def serve(self, timeout: int = 0):
+        signal.signal(signal.SIGINT, sigint_handler)
+        threading.current_thread().shutdown_flag = False
```

Comment:
This shutdown flag is a custom variable? I don't seem to see it in the threading class

Reply:
Yeah we're using this to flag shutdowns. The default sigint handling kills the gRPC connection and we don't want that to happen until user intent is confirmed.